### DISCUSSION TITLE
Wikijs Installation on Synology

### DISCUSSION THREAD

**Discussion by kwasposer** [requarks/wiki#7393] [Help / Questions]
Hello, can you guide me step by step how to setup Wikijs on my Snology NAS DS218+? 

Thank you sooo much!

**Comment by BPowell76** [comment]
If you're able to get Docker on it (I didn't see it on their list of packages, but maybe you can get it third-party), you should be able to setup Wikijs via Docker containers ([install steps](https://docs.requarks.io/install/docker))

**Reply by kwasposer** [reply]
Thanks, i did install Docker but still struggle. There is an wikijs Docker available. Some people use mysql , and some use postgres. I am but confused what is state of art.

**Reply by BPowell76** [reply]
You can use either while Wiki.js is version 2.x, but when version 3.0 releases only PostgreSQL will be supported.
